### PR TITLE
Support More Platform-bridge Payments Method

### PR DESCRIPTION
**Pull request by yusoofsh**
Please review and if there's question, please don't hesitate to ask.

### CODE REVIEW

File: lib/flutrans.dart [hunk 1]
```diff
@@ -36,21 +36,13 @@ class Flutrans {
...
   }
 
-  Future<void> init(String clientId, String url,
-      {String env = 'production'}) async {
-    await _channel.invokeMethod("init", {
-      "client_key": clientId,
-      "base_url": url,
-      "env": env,
-    });
+  Future<void> init(String clientId, String url) async {
```
Comment (apinprastya) [inline lib/flutrans.dart:39]: env works on IOS, not sure why you remove it?

File: lib/flutrans.dart [hunk 2]
```diff
@@ -104,20 +102,28 @@ class MidtransTransaction {
...
+    this.customField1,
+    this.customField2,
+    this.customField3,
   });
+
   Map<String, dynamic> toJson() {
     return {
       "total": total,
-      "skip_customer": skipCustomer,
+      "customer": customer,
```
Comment (apinprastya) [inline lib/flutrans.dart:121]: Customer is an object, this will not work

File: lib/flutrans.dart [hunk 3]
```diff
@@ -127,12 +133,8 @@ class TransactionFinished {
...
   final String statusMessage;
-  final String response;
-  TransactionFinished(
-    this.transactionCanceled,
-    this.status,
-    this.source,
-    this.statusMessage,
-    this.response,
-  );
+  final Map<dynamic, dynamic> response;
```
Comment (apinprastya) [inline lib/flutrans.dart:136]: Use map<String, dynamic>

File: lib/flutrans.dart (hunk 1 above)
Comment (yusoofsh) [reply lib/flutrans.dart:39]: my mistakes, I accidentally remove that lines.

File: lib/flutrans.dart (hunk 3 above)
Comment (yusoofsh) [reply lib/flutrans.dart:136]: definitely. I'll fix that.

File: lib/flutrans.dart (hunk 2 above)
Comment (yusoofsh) [reply lib/flutrans.dart:121]: thank you!

**Comment by yusoofsh** [comment]
Sorry for the inconvenient, closing it in favor of webview.
Thank you.

**Pull request closed without merging**